packages/calcite-components/src/assets/styles/includes.scss

```diff
@@ -75,6 +71,16 @@
   }
 }
 
+@mixin calciteComponentCommon() {
```

Comment:
We don't prefix other mixins w/ `calciteComponent`. Maybe `base` or `common`?

Reply:
I also noticed some mixins use camel case vs dash separated? do we want to pick one?

- calciteHydratedHidden
- xButton
- hidden-form-input
- word-break

Reply:
Renamed to `baseComponent`